omdb/models.py

```diff
@@ -80,6 +80,7 @@ class Item(Storage):
         'Runtime',
         'Season',
         'seriesID',
+        'totalSeasons',
```

Comment:
My suggestion is to add `totalSeason` in this PR. 
I hope I'll find time on the weekend to fix the `totalResults`. 
I hope I'll also fix  the  #13  issue.